app/controllers/host_shelters/properties_controller.rb

```diff
@@ -0,0 +1,24 @@
+class HostShelters::PropertiesController < HostShelters::HostSheltersController
+  before_action :auth_user_check?
+
+  def new
+    @property = Property.new
+  end
+
+  def create
+  property = Property.create(safe_params)
+   binding.pry
+   if property.save
```

Comment:
I noticed the property won't save without an associated user, maybe shovel the new property into current_user or specify current_user in the create?

Reply:
this just clicked with me, I got hung up here and didnt' finish this method, because I was trying to add the auth_user_check? method to verify I had a user to associate this with before they could sign up?

Looking back on it, is that just a lazy way of doing it? Would throwing in @jtruong2 manual user creation form as a possible conditional be better or more complicated??